.github/workflows/kind-e2e.yaml

```diff
@@ -66,12 +66,7 @@ jobs:
         # Build Knative plugin
         go build -o kn-vsphere ./plugins/vsphere/cmd/vsphere
 
-        kubectl -n vmware-sources wait --timeout=10s --for=condition=Available deploy/vsphere-source-webhook
```

Comment:
Maybe have this commented or in the docs for new users to know where to look?

Reply:
I added a comment why this was bumped to `1m`.